source/__tests__/integration/export-profile.test.js

```diff
@@ -89,4 +93,104 @@ describe('Profiles Export', () => {
       expect(json['Profile']['resourceTemplates'][0]['propertyTemplates'][0]['propertyURI']).toEqual(propURI)
     })
   })
+
+  describe('schema version', () => {
```

Comment:
showing the schema version set upon export seems like an important documentation of behavior.  Note that it is currently always 0.0.9.  It can't be a later version because the profile editor outputs a lot of empty values